x/params/space/test_common.go

```diff
@@ -0,0 +1,40 @@
+package space
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	abci "github.com/tendermint/tendermint/abci/types"
+	dbm "github.com/tendermint/tendermint/libs/db"
+	"github.com/tendermint/tendermint/libs/log"
+
+	"github.com/cosmos/cosmos-sdk/store"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/wire"
+)
+
+// Keys for parameter access
+const (
+	TestParamSpace = "ParamsTest"
+)
+
+// Returns components for testing
+func DefaultTestComponents(t *testing.T) (sdk.Context, Space, func() sdk.CommitID) {
```

Comment:
Do you intend for this to be used in other modules? If not please make these unexported.

Reply:
It is used by the submodules, and also can be used for modules those uses paramstore internally.